keyboards/gboards/gergoplex/config.h

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2013 Oleg Kostyuk <cub.uanic@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// Copy and worked on with love from the EZ team
+
+#pragma once
+#include "config_common.h"
+
+#define BOOTMAGIC_LITE_ROW 9
+#define BOOTMAGIC_LITE_COLUMN 2
+
+#define VERBOSE
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x0007
```

Comment:
Surprisingly, not used, but not exactly comfortable about a VID this low.

Reply:
Changed to `6B0A` (GBOArds)